app/main.py

```diff
@@ -2,5 +2,13 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    store_results = {}
+
+    def inner(*args) -> str:
+        if args in store_results.keys():
+            print("Getting from cache")
+        else:
+            store_results.update({args: func(*args)})
```

Comment:
.update is redundant here
```suggestion
            store_results[args] = func(*args)
```
Simplify and use brackets